data/features/jupyterLab/jupyterLab.dat.sh

```diff
@@ -10,7 +10,10 @@ jupyterLab_binariesinstalledpaths=("bin/jupyter-lab;jupyter-lab" "bin/jupyter;ju
 jupyterLab_packagedependencies=("libkrb5-dev")
 jupyterLab_flagsoverride=";;1;;;"  # Ignore Errors to check dependencies. This is a patch
 jupyterLab_launcherkeynames="defaultLauncher"
-jupyterLab_defaultLauncher_exec="jupyter-lab &"
+jupyterLab_defaultLauncher_exec="bash ${CURRENT_INSTALLATION_FOLDER}/jupyterLab.sh"
```

Comment:
I think that you should do a link jupyterLab.sh as the command jupyter and use this as the command to call in the desktop launcher.

Reply:
jupyter is not the same as jupyter-lab

Reply:
Well, the point is that you should use a link. I do not care how do you call it.

Reply:
@Axlfc